lib/mix/tasks/phoenix.new.ex

```diff
@@ -33,6 +34,12 @@ defmodule Mix.Tasks.Phoenix.New do
 
     copy_from template_dir, path, application_name, &EEx.eval_file(&1, binding)
     copy_from static_dir, Path.join(path, "priv/static"), application_name, &File.read!(&1)
```

Comment:
We still need to copy the "built" app.js boilerplate because it bundles phoenix.js, which live-reload needs. Alternatively, we can have the injected scripts point to a different priv/static/js/phoenix.js file, but then this breaks our vendor idioms, so I chose to build the bundle into phoenix's priv/static and still copy over.